bot/cogs/timeleft.py

```diff
@@ -0,0 +1,23 @@
+from discord.ext import commands
+from dateutil.relativedelta import relativedelta
+import logging
+import aiohttp
+import discord
+import datetime
+
+bot = commands.Bot(command_prefix='.', description='pr bot')
+

```

Comment:
The commands for this cog should be methods of a class, whose `__init__` method accepts and stores the bot instance. There should also be a global `setup` function defined to accept this bot instance and instantiate the class. Check out the source for some of our other cogs (like [`hacktoberstats`](https://github.com/discord-python/hacktoberbot/blob/master/bot/cogs/hacktoberstats.py)) for the general layout.

See also the [`load_extension`](https://discordpy.readthedocs.io/en/rewrite/ext/commands/api.html#discord.ext.commands.Bot.load_extension) documentation.